index.html

```diff
@@ -149,7 +151,7 @@ <h2>Scope</h2>
             <a href="https://www.w3.org/TR/did-spec-registries/">DID Specification Registries</a>, including transitioning it from a WG Note to a W3C Registry, should such a formal registry be defined by W3C Process and should that process allow such a transition.
           </li>
           <li>
-            Decentralized Characteristics Rubric
+            <a href="https://www.w3.org/TR/did-rubric/">Decentralized Characteristics Rubric</a> <i class=todo>(Note: this link is currently 404, the planned publication for this note is mid-August)</i>
```

Comment:
The title of the rubric changed due to the expansion of scope beyond just decentralization.
```suggestion
            <a href="https://www.w3.org/TR/did-rubric/">DID Method Rubric</a> <i class=todo>(Note: this link is currently 404, the planned publication for this note is mid-August)</i>
```